kustomize/install/default/manager-labels-target.yaml

```diff
@@ -0,0 +1,5 @@
+- op: remove
+  path: /spec/selector/matchLabels/app.kubernetes.io~1name
+
+- op: remove
+  path: /spec/selector/matchLabels/app.kubernetes.io~1version
```

Comment:
I'm curious why these are being removed

Reply:
We add the version as a common label and change it with each release, but it's not good to change a Deployment selector like this. Is there another way to tell Kustomize not to change selectors when using [`commonLabels`](https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/commonlabels/)?